tests/settings/e2e/features/stakepool-settings.feature

```diff
@@ -0,0 +1,34 @@
+@e2e @watch
+Feature: Stake-pool server settings
+
+  Background:
+    Given I have completed the basic setup
+    And I have the following wallets:
+      | name        |
+      | Test Wallet |
+
+  Scenario: Navigating to Stake-pool settings screen and selecting custom stake-pool server
+    Given I am on the General Settings "stake-pools" screen
+    When custom server is the default option
+    And The smash server input textBox is visible
+    And I enter custom server "https://smash.cardano-testnet.iohkdev.io/" as the custom server option
+    And I click on Daedalus logo to change focus
+    And I click the stake-pool custom server input box submit button
+    And I see the your changes have been saved success message
```

Comment:
You can declare a message like "your changes have been saved" and check if that is correct.
e.g. And I see the "Your changes have been saved" success message